build.gradle.kts

```diff
@@ -1,6 +1,5 @@
-/*
- * This file was generated by the Gradle 'init' task.
- *
- * This is a general purpose Gradle build.
- * Learn how to create Gradle builds at https://guides.gradle.org/creating-new-gradle-builds
- */
+apply(from = "gradle/build-scan-data.gradle")
+
+tasks.register("quickCheck") {
+    dependsOn(":worblehat-domain:check", ":worblehat-web:check")
```

Comment:
:worblehat-web:check includes the WorblehatSystemIT integration test, which should not run during unit tests. In maven it was run as Integration Test with the failsafe plugin.

Reply:
Good catch. I can add some additional configuration to that `WorblehatSystemIT` is executed by a dedicated task that is not part of `check`. However I wonder what's the value of executing this test in it's own stage. Executing the test only takes 11ms on my machine: https://scans.gradle.com/s/sclv3j4sdmd56/tests#:worblehat-web-test-de.codecentric.psd-worblehatsystemit

Reply:
Okay, I ran the tests again and `WorblehatSystemIT` adds 10s of execution time. I'll add some configuration that will exclude it.

Reply:
@AndreasEK I've excluded `WorblehatSystemIT` from `quickTest` (I renamed it from `quickCheck` to `quickTest` which felt more appropriate).